doc/run_on_gcp.md

```diff
@@ -0,0 +1,169 @@
+
+1. Create a CloudSQL MySQL instance.
+    ``` bash
+    > gcloud beta sql instances create sqlflow-cloudsql-instance --network sqlflow-demo-vpc --no-assign-ip --tier db-n1-standard-2 --region us-central1
+    Creating Cloud SQL instance...done.
+    NAME                       DATABASE_VERSION  LOCATION       TIER              PRIMARY_ADDRESS  PRIVATE_ADDRESS  STATUS
+    sqlflow-cloudsql-instance  MYSQL_5_7         us-central1-a  db-n1-standard-2  -                10.20.0.3        RUNNABLE
+    ```
+
+    You can find the *Instance connection name* and *Private IP address* for the
+    newly created CloudSQL instance at the [CloudSQL console](https://console.cloud.google.com/sql/instances/sqlflow-cloudsql-instance/overview).
+    Note that
+    - the private ip address is within the IP range we reserved in the previous step.
+    - You should be able to see a private connection between your VPC and the
+      newly created CloudSQL instance on the VPC console.
+
+1. Configure the default CloudSQL MySQL user account password
+   For this demo purpose, we assume the *root* user of our MySQL instance has
+   password *root*. To configure this, run the following:
+    ``` bash
+    > gcloud sql users set-password root --host=% --instance=sqlflow-cloudsql-instance --password=root
+    ```
+
+
+## Set Up a Kubernetes Cluster on Google Kubernetes Engine
+
+1. Create a VPC subnet to be used by our Kubernetes cluster on GKE.
+    ``` bash
+    > gcloud compute networks subnets create sqlflow-demo-subnet --network sqlflow-demo-vpc --region us-central1 --range 10.10.0.0/16
+    NAME                 REGION       NETWORK           RANGE
+    sqlflow-demo-subnet  us-central1  sqlflow-demo-vpc  10.10.0.0/16
+    ```
+
+1. Create a Kubernetes cluster on GKE.
+    ``` bash
+    > gcloud container clusters create sqlflow-gke-cluster --zone us-central1-a --num-nodes 3 --network sqlflow-demo-vpc --subnetwork sqlflow-demo-subnet
+    kubeconfig entry generated for sqlflow-gke-cluster.
+    NAME                 LOCATION       MASTER_VERSION  MASTER_IP    MACHINE_TYPE   NODE_VERSION   NUM_NODES  STATUS
+    sqlflow-gke-cluster  us-central1-a  1.12.8-gke.10   108.50.81.7  n1-standard-1  1.12.8-gke.10  3          RUNNING
+    ```
+
+    Note that after this, gcloud will automatically add the proper
+    configurations for kubectl CLI. You can verify that kubectl is configured properly by running
+    ``` bash
+    > kubectl config current-context
+    ```
+    It should show a cluster name like 'gke_${YOUR_GCP_PROJECT_ID}'-central1-a_sqlflow-gke-cluster
+
+
+## Deploy SQLFlow services
+
+You can refer to the [Running SQLFlow on Kubernetes](/doc/run_on_kubernetes.md)
+tutorial for a more comprehensive guide on deploying SQLFlow on Kubernetes
+cluster.
+
+In this tutorial, the difference is that we are relying on an existing MySQL
+instance (hosted on CloudSQL). We will show how to deploy a miminal SQLFlow
+service (SQLFlow server and Jupyter Notebook server) below.
```

Comment:
miminal => minimal

I'm not sure `miminal SQLFlow ` is exactly enough, how about **deploy SQLFlow gRPC server and Jupyter Notebook server** ?

Reply:
SG. Done :)